Source/Base/FieldContainer/Base/OSGFieldContainer.cpp

```diff
@@ -380,7 +382,7 @@ void FieldContainer::registerChangedContainer(void)
 
     pCL->addUncommited(_pContainerChanges);
 
-    osgSpinLockRelease(&_uiContainerId, SpinLockClearMask);
+    //osgSpinLockRelease(&_uiContainerId, SpinLockClearMask);
```

Comment:
Maybe the same note here about the lock already being acquired.

Reply:
Done